package.json

```diff
@@ -19,5 +21,9 @@
     "express": "^4.17.1",
     "node-fetch": "^2.6.1",
     "nodemon": "^2.0.7"
+  },
+  "devDependencies": {

```

Comment:
Nodemon should probably be a dev dependency. This is another oof from when I first prototyped this before putting it on github. I just added nodemon without caring.